src/ayab/encoders.h

```diff
@@ -63,6 +63,9 @@ constexpr uint8_t END_LEFT[NUM_MACHINES] = {0U, 0U, 0U};
 constexpr uint8_t END_RIGHT[NUM_MACHINES] = {255U, 255U, 140U};
 constexpr uint8_t END_OFFSET[NUM_MACHINES] = {28U, 28U, 5U};
 
+constexpr uint8_t END_LEFT_PLUS_OFFSET[NUM_MACHINES] = {28U, 28U, 5U};
```

Comment:
The name of the variable tells already, but to be sure, please add a comment explaining that this is just the sum/difference of the values above to save an addition during ISR.

Reply:
Good suggestion.